spec/cqueues_spec.lua

```diff
@@ -130,4 +161,52 @@ describe("lredis.cqueues module", function()
 		assert(cq:loop(1))
 		assert(cq:empty())
 	end)
+	it("has working connect constructor", function()
+		local m = cs.listen{host="127.0.0.1", port="0"}
+		local _, host, port = m:localname()
+		local cq = cqueues.new()
+		cq:wrap(function()
+			local r = lc.connect("redis://@password:localhost:"..port.."/5")
+			assert.same(r:ping(), "PONG")
+			r:close()
+		end)
+		cq:wrap(function()
+			local s = m:accept()
+			interact(s, {
+				{ read=true, "*2", "$4", "AUTH", "$8", "password" },
```

Comment:
Oops. that should have been a `testInteraction()`, which I extracted from that test and hadn't updated it.